cfgov/ask_cfpb/views.py

```diff
@@ -6,6 +6,12 @@
 def view_answer(request, slug, language, answer_id):
     answer_page = get_object_or_404(
         AnswerPage, language=language, answer_base__id=answer_id)
+    if answer_page.redirect_id:
+        new_page = get_object_or_404(
+            AnswerPage,
+            language=language,
+            answer_base__id=answer_page.redirect_id)
+        return redirect(new_page.url)
     if "{}-{}-{}".format(slug, language, answer_id) != answer_page.slug:
```

Comment:
Minor thing in Python2 - if slug has unicode character this will raise an exception unless you mark the string with `u"{}-{}-{}"` or include `from __future__ import unicode_literals`.